src/prommis/nanofiltration/README.md

```diff
@@ -0,0 +1,24 @@
+# membrane examples
```

Comment:
I only see an `index.md` in the docs folder, should I create a new file `index.rst`? Also, does the text from this `README.md` file become the `index.rst`, or would I have a separate file for this, like `membrane_examples.rst`?

Reply:
@lbianchi-lbl Is probably the one to answer this - I am not sure if we have decided how the docs will be written yet (I just presumed it would be the same as IDAES).

Reply:
That's a good question. I think that, for the time being (i.e. for the March release), this README would be just fine. Once the Sphinx harness is properly set up, then we can move the content appropriately.